views/layout.jade

```diff
@@ -1,11 +1,11 @@
-!!! 5
+doctype html
 html
   head
     title #{settings.sitename} - #{title}
     link(rel='stylesheet', href='/stylesheets/bootstrap.min.css')
-    style(type='text/css')
-      body{padding-top:60px;}
+    link(rel='stylesheet', href='/stylesheets/custom.css')
```

Comment:
In which case the better solution would be to actually use a CSS pre-processor that builds SCSS or LESS files into a combined, minified, uglified CSS file, as part of a gulp or grunt build process. 

I may as well add this to my TODO file for this app.